e2e-tests/portal/cypress/e2e/journeys/maker/abandon-a-deal/abandon-a-deal.spec.js

```diff
@@ -1,57 +1,51 @@
-const { contract, contractDelete, defaults } = require('../../../pages');
+const { contract, contractDelete, defaults, dashboardDeals } = require('../../../pages');
 const { successMessage } = require('../../../partials');
-const relative = require('../../../relativeURL');
 const MOCK_USERS = require('../../../../../../e2e-fixtures');
-const twentyOneDeals = require('../../../../fixtures/deal-dashboard-data');
+const { additionalRefName, bankInternalRefName } = require('../../../../fixtures/deal');
 
 const { BANK1_MAKER1, ADMIN } = MOCK_USERS;
 
 context('A maker selects to abandon a contract from the view-contract page', () => {
-  let deal;
-
   before(() => {
-    const aDealInStatus = (status) => twentyOneDeals.filter((aDeal) => status === aDeal.status)[0];
-
     cy.deleteDeals(ADMIN);
-    cy.insertOneDeal(aDealInStatus('Draft'), BANK1_MAKER1).then((insertedDeal) => {
-      deal = insertedDeal;
-    });
+    cy.createBssEwcsDeal({});
   });
 
   it('The cancel button returns the user to the view-contract page.', () => {
     // log in, visit a deal, select abandon
-    cy.login(BANK1_MAKER1);
-    contract.visit(deal);
+    cy.loginGoToDealPage(BANK1_MAKER1);
+
     contract.abandonLink().contains('Abandon');
     contract
       .abandonLink()
       .invoke('attr', 'aria-label')
       .then((label) => {
-        expect(label).to.equal(`Abandon Deal ${deal.bankInternalRefName}`);
+        expect(label).to.equal(`Abandon Deal ${bankInternalRefName}`);
```

Comment:
this `bankInternalRefName` currently has nothing to do with the deal being created - I've left a comment on e2e-tests\portal\cypress\support\portal\createBssEwcsDeal.js that would fix this

Reply:
To be checked.